server/api/game/controller.js

```diff
@@ -94,22 +161,48 @@ const moveGhosts = ({ ghosts, gridState, scatterStart }) => {
 
 const calculateNextGameState = () => {
   const {
-    ghosts, gridState, scatterStart,
+    ghosts, gridState, scatterStart, pacmanOne, pacmanTwo,
   } = gameState;
-  const { ghostsUpdated, moveGhostsCount } = moveGhosts(
-    { ghosts, gridState, scatterStart },
-  );
+  const { ghostsUpdated, moveGhostsCount } = moveGhosts({ ghosts, gridState, scatterStart });
+  const { pacmanUpdated: newPacmanOne } = movePacman({
+    pacman: pacmanOne, ghostsUpdated, gridState,
+  });
+  const { pacmanUpdated: newPacmanTwo } = movePacman({
```

Comment:
The code is repetitive over here. Make a function to deal with two Pacman Objects